chicagoland.pandemicresponsecommons.org/manifests/scaling/scaling.json

```diff
@@ -0,0 +1,57 @@
+{
+  "arborist": {
+    "min": 1,
```

Comment:
these are not the right values, see the current manifest

Reply:
This is being copied from qa-covid19... 🤷🏼‍♂️ 
We can suppress this copy though.